webapp/src/js/controllers/inbox.js

```diff
@@ -219,12 +222,21 @@ var _ = require('underscore'),
         Telemetry.record('boot_time', dbWarmed - $window.startupTimes.start);
 
         delete $window.startupTimes;
+        initSecondaryServices();
```

Comment:
Secondary "Services" is maybe a strange term? Maybe "lazyloadContent" or "loadContentAfterBootstrap" or alike?

Reply:
Updated to: `lazyLoadTasks`